src/Trinity.C/src/Storage/MemoryTrunk/MemoryTrunk.Memory.cpp

```diff
@@ -177,7 +177,7 @@ namespace Storage
 
         temp_head_group.append_head = (uint32_t)ReloadImpl(); // Update hashtable.CellEntries
 
-        InfoLog("Memory trunk {0} reloaded.", TrunkId);
+        InfoLog(String("Memory trunk {0} reloaded."), TrunkId);
```

Comment:
Discussed with Leo: the two signatures of WriteLine are ambiguous as char* could be implicitly casted to String. So here we explicitly cast it to String first, so as for this call to fall into the perfectly matching method.